### DISCUSSION TITLE
vike-react SSG with dynamic page not functioning when using netlify

### DISCUSSION THREAD

**Discussion by muhrifqii** [vikejs/vike#2036] [Discussion] [answered]
Hello, I'm not familiar with the render mode so I used vike-react on my personal site. I read the doc and the example that we can add Parameterized routes by using `@`. I tried locally and all fine and the page successfully load. I also use the `vite build && vite preview` to test the final build result, and all seems fine, no issue. But after I deployed it on netlify, the page wont load and redirect to my 404 page.

My page structure looks like this:
![image](https://github.com/user-attachments/assets/1d057384-fa33-4276-ba37-3c46a561c099)

for SSG, using dist/client is the correct path right? Or I need the whole dist folder? I already successfully deployed it but without any routes other than `/`

**Comment by brillout** [comment, marked as answer]
https://github.com/vikejs/vike/issues/1476

**Reply by muhrifqii** [reply]
Thanks for connecting the dot, I will mark this as answered and continue on the github issue you mentioned
